core/db/entities.py

```diff
@@ -296,11 +306,12 @@ class Programme(BaseModel):
     programme_id = sqla.Column(sqla.String(), nullable=False, unique=True)
 
     programme_name = sqla.Column(sqla.String(), nullable=False)
-    fund_type_id = sqla.Column(sqla.String(), nullable=False)
     organisation_id = sqla.Column(GUID(), sqla.ForeignKey("organisation_dim.id"), nullable=False)
+    fund_type_id = sqla.Column(GUID(), sqla.ForeignKey("fund_dim.id"), nullable=False)
```

Comment:
This gets particularly confusing because now the FK related fields are called `fund_type_id`, and connect to the `fund_dim` table which has a `fund_type_id` field - but that isn't the field we're talking about.

I think I'd either call these `fund_id` or `fund_dim_id`? Possibly updating the `Fund` python model to `FundDim` so that we're using consistent terminology.

Reply:
I've changed the name of the column in ```fund_dim```, I'd propose leaving this as ```fund_type_id``` for now in the Programme table as during ingestion/transformation it is ```FundType_ID```, and is then serialised out as ```FundTypeID``` so may just add extra confusion if the 'type' bit disappears between these two processes. We can of course talk to the business about serialising out the name as something different in the future.

Reply:
Ah interesting - good context - thanks.